src/packaging/version.py

```diff
@@ -221,6 +225,45 @@ def __init__(self, version: str) -> None:
             self._version.local,
         )
 
+    def replace(
+        self,
+        release: Tuple[int, ...] = _SENTINEL,
+        pre: Optional[Tuple[str, int]] = _SENTINEL,
+        post: Optional[int] = _SENTINEL,
+        dev: Optional[int] = _SENTINEL,
+        local: Optional[str] = _SENTINEL,
+    ) -> "Version":
```

Comment:
```suggestion
    def replace(
        self,
        *,
        release: Tuple[int, ...] = _SENTINEL,
        pre: Optional[Tuple[str, int]] = _SENTINEL,
        post: Optional[int] = _SENTINEL,
        dev: Optional[int] = _SENTINEL,
        local: Optional[str] = _SENTINEL,
    ) -> "Version":
```

Let’s make these keyword-only, I don’t think using those as positional makes sense.